rosetta/patchlist-t5x.txt

```diff
@@ -7,4 +7,4 @@
 
 pull/1392/head  # https://github.com/google-research/t5x/pull/1392: Add support for partial checkpoint restore
 pull/1393/head  # https://github.com/google-research/t5x/pull/1393: Adds DALI support to t5x
-pull/1391/head  # https://github.com/google-research/t5x/pull/1385: Adds transformer engine support and GPU optimizations to T5x (enables H100)
+mirror/pull/5/head # [REBASED] https://github.com/google-research/t5x/pull/1385: Adds transformer engine support and GPU optimizations to T5x (enables H100)
```

Comment:
Can this be reverted? As of yesterday, 1391 builds successfully now

Reply:
Definitely. Thanks for the reminder